WordPress/gutenberg
# Removes margin right from wp-block-tag-cloud

## What?
When you set a Tag Cloud block to the "Outline" style, the spacing between the items is set both with a gap on the container and a margin-right on reach individual link element, resulting in uneven spacing between the items.
Resolves https://github.com/WordPress/gutenberg/issues/66678

## Why?
.wp-block-tag-cloud a has margin right property causing this.

## Testing Instructions
1. Insert a Tag Cloud block.
2. Set it to the "Outline" style.
3. Inspect it on the front-end, and verify that there's both a gap on the .wp-block-tag-cloud container and a margin-right on each link element, resulting in an uneven spacing between each item.

### Testing Instructions for Keyboard


## Screenshots or screencast 
<img width="775" alt="Screenshot 2024-11-13 at 9 45 20 PM" src="https://github.com/user-attachments/assets/73ddec9c-9cf6-4904-ae2f-c2b02b058a5f">
